### PR TITLE
configure to support addon manager installation

### PR DESCRIPTION
**Pull request by sliptonic**

### CODE REVIEW

File: package.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
+<package format="1" xmlns="https://wiki.freecad.org/Package_Metadata">
+  <name>Ondsel Lens</name>
+  <description>Workspace Manager for Ondsel and Local Workspaces</description>
+  <version>0.1.0</version>
+  <date>2023-07-06</date>
+  <maintainer email="development@ondsel.com">Shai Seger</maintainer>
```
Comment (amrit3701) [inline package.xml:7]: Is maintainer name **Shai Seger** intentional?

Comment (sliptonic) [reply package.xml]: Nope.  Oversight.  Fixed.

**Comment by PaddleStroke** [comment]
You can then remove the z_Package.xml file

**Comment by PaddleStroke** [comment]
I am no expert of package.xml but it looks good.

**Pull request merged**